.gitignore

```diff
@@ -11,3 +11,4 @@
 !.vscode/settings.json
 node_modules/
 *.log
+force-app/main/default/staticresources/vanilla/main.js
```

Comment:
This file is now generated. To produce it, run `yarn deploy` from `packages/vanilla-datatable`. A future PR will introduce lerna and a global build/deploy step.